fmriprep/cli/run.py

```diff
@@ -77,6 +77,8 @@ def get_parser():
     #                     help='select a specific run to be processed')
     g_bids.add_argument('-t', '--task-id', action='store',
                         help='select a specific task to be processed')
+    g_bids.add_argument('--echo-idx', action='store',
```

Comment:
`type=int,`?

And maybe let's add the short option `'-e'`.

Reply:
The `-e` option is conflicting with the `-e` option for setting the env in `fmriprep-docker`. I'm reverting to only the long-form command, but let me know if you'd prefer a different approach !

Reply:
That seems fine. We can always reconsider later, but I agree, let's not cause a conflict.